executorlib/standalone/cache/spawner.py

```
@@ -24,7 +27,11 @@ def execute_in_subprocess(
         task_dependent_lst = [
             task for task in task_dependent_lst if task.poll() is None
         ]
-    return subprocess.Popen(command, universal_newlines=True, cwd=cwd)
+    if resource_dict is None:
+        resource_dict = {"cwd": None}
+    elif len(resource_dict) == 0:
+        resource_dict = {"cwd": None}
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)